src/styles/components/_select.scss

```diff
@@ -0,0 +1,3 @@
+.c-select__menu {
```

Comment:
Dit mag naar `components/Select/Select.scss`

Reply:
Deze gaat ook een hogere value moeten hebben want met 3 gaat die nog altijd niet over de buttons vd WYSIWYG

Reply:
> Dit mag naar `components/Select/Select.scss`

Ik denk dat het beter daar blijft aangezien we ook rechtstreeks de react-select gebruiken in de TagInput component

Reply:
z-index is geupdate naar 12:
![image](https://user-images.githubusercontent.com/1710840/74145730-d45f9900-4bff-11ea-9c00-5f88a1d45ed3.png)

Reply:
In de TagsInput component krijgt die nergens de class `c-select` mee?

Reply:
goed punt
ik dacht dat die class rechtstreeks van de react-select package kwam
in dat geval is het idd logischer dat de styles bij de components zitten

fixed